kubernetes/config/samples/imagenet.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: elastic.pytorch.org/v1alpha1
+kind: ElasticJob
+metadata:
+  name: imagenet
+  namespace: elastic-job
+spec:
+  rdzvEndpoint: "etcd-service:2379"
+  minReplicas: 1
+  maxReplicas: 5
+  replicaSpecs:
+    Worker:
+      replicas: 3
+      restartPolicy: ExitCode
+      template:
+        apiVersion: v1
+        kind: Pod
+        spec:
+          containers:
+            - name: elasticjob-worker
+              image: seedjeffwan/examples:0.1.0rc1
```

Comment:
this should use the one from torchelastic as you did above for classy vision?

Reply:
ah. nice cache, I forget to update it back. BTW, any latest version available recently?